Add answers

samamaryna opened the pull request:
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Add answers to the answers in the assignment. 
## What did you learn from the changes you have made?
Practised creating the survey. 
## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?

## How were these changes tested?

## A reference to a related issue in your repository (if applicable)

## Checklist
- [X] I can confirm that my changes are working as intended

samamaryna committed:
Add answers

KasraVakiloroayaei commented:
A2: INCOMPLETE
Please specify the sampling strategy and justify it (SRS/STRATA/CLUSTER, etc...)
I would also avoid open ended questions on a questionnaire as the follow up analysis on such questions can be very difficult.

samamaryna committed:
Update answers, add sampling strategy

samamaryna commented:
> A2: INCOMPLETE Please specify the sampling strategy and justify it (SRS/STRATA/CLUSTER, etc...) I would also avoid open ended questions on a questionnaire as the follow up analysis on such questions can be very difficult.

Added sampling strategy and changed questions.

KasraVakiloroayaei commented:
A2: PASS 
Great work!
